inception-project/inception
# #2678 - Set default annotation editor in project settings

**What's in the PR**
- Added project-level preference for the annotation editor. If the manager sets it, the user's preferred editor is ignored.
- Move project initializers to a separate module

**How to test manually**
* Try setting a editor in the project settings and observe that that is the one used on the annotation page

**Automatic testing**

**Documentation**
